btree/btree.c

```diff
@@ -2876,14 +2910,12 @@ static int64_t btree_put_tick(struct m0_sm_op *smop)
 			return P_COOKIE;
 		else
 			return P_SETUP;
-	case P_COOKIE: {
-		void *cookie_segaddr = bop->bo_rec.r_key.k_cookie.segaddr;
+	case P_COOKIE:
 		if (cookie_is_valid(tree, &bop->bo_rec.r_key.k_cookie) &&
-		    !cookie_overflow_is_possible(cookie_segaddr))
+		    !node_overflow(oi->i_cookie_node))
```

Comment:
Currently, passing oi->i_cookie_node to node_overflow function. But memory for oi is not yet allocated , we may allocate memory in previous phases. As cookie implementation is yet to be discussed , we may need to revisit to this section when we will implement cookies.